index.js

```diff
@@ -42,9 +29,43 @@ module.exports = {
       parent = parent.app;
     }
 
-    importFontAwesome(parent);
+    let fontAwesomeToImport = fs.readdirSync(`${this.nodeModulesPath}/font-awesome/fonts`);
+    fontAwesomeToImport.forEach((fontFileName) => {
+      parent.import(path.join('vendor', 'fonts', fontFileName));
+    });
+
     importFixtable(parent);
 
     parent.import('vendor/styles/fixtable-ember.css');
+  },
+
+  treeForVendor(tree) {
+    let treesToMerge = [];
+    if (tree) {
+      treesToMerge.push(tree);
+    }
+
+    let fontAwesomeFunnel = new Funnel(`${this.nodeModulesPath}/font-awesome`, {
+      srcDir: 'fonts',
+      destDir: 'fonts'
+    });
+    treesToMerge.push(fontAwesomeFunnel);
+
+    return mergeTrees(treesToMerge, { overwrite: true });
+  },
+
+  treeForStyles(tree) {
+    let treesToMerge = [];
+    if (tree) {
+      treesToMerge.push(tree);
+    }
+
+    let fontAwesomeCssFunnel = new Funnel(`${this.nodeModulesPath}/font-awesome`, {
+      srcDir: 'css',
```

Comment:
wondering here if I should just use `files: ['font-awesome.css']` since there are both the css, the .map, and the min.css files in this directory.  It works as-is but I suspect both css files get concatenated to the vender.css